README.md

```diff
@@ -1,2 +1,2 @@
 # Blog project
-This is a Node, Mongo and Express project with a simple blog. More info to come soon.
+This is a Node, MongoDB, EJS and Express project with a simple blog. The /compose route publishes a new blog to / route. More info to come soon.
```

Comment:
I believe this could say 'This is a Node, MongoDB, EJS and Express project with a simple blog. The /compose route publishes a new blog *post* to /route  *and stores it on the database*'